modin/pandas/dataframe.py

```diff
@@ -45,9 +56,7 @@
 from .groupby import DataFrameGroupBy
 
 
-@_inherit_docstrings(
-    pandas.DataFrame, excluded=[pandas.DataFrame, pandas.DataFrame.__init__]
```

Comment:
`pandas.DataFrame` is removed from the excluded scope as class docstring is not defined by us.